src/main/java/org/cryptomator/cryptofs/health/dirid/OrphanDir.java

```diff
@@ -33,5 +58,119 @@ public String toString() {
 		return Map.of(ENCRYPTED_PATH, dir.toString());
 	}
 
-	// fix: create new dirId inside of L+F dir and rename existing dir accordingly.
+	@Override
+	public void fix(Path pathToVault, VaultConfig config, Masterkey masterkey, Cryptor cryptor) throws IOException {
+		var sha1 = getSha1MessageDigest();
+		String runId = BaseEncoding.base64Url().encode(sha1.digest(UUID.randomUUID().toString().getBytes(StandardCharsets.UTF_8))).substring(0, 3);
```

Comment:
no need to sha1 the UUID
```suggestion
		String runId = Integer.toString((short) UUID.randomUUID().getMostSignificantBits(), 32);
```